galaxy/api/views/namespace.py

```diff
@@ -258,7 +258,7 @@ def update(self, request, *args, **kwargs):
                 owners.append(request.user.pk)
             update_owners(instance, owners)
 
-        for item in ('name', 'description', 'avatar_url', 'location', 'company', 'email', 'html_url', 'active'):
+        for item in ('name', 'description', 'avatar_url', 'location', 'company', 'email', 'html_url', 'active', 'vendor'):
```

Comment:
Probably we shouldn't update `vendor` field from API. It is only to be managed by admin users.

Reply:
By 'admin' we mean user.is_staff=True, not the Django admin. I think what @alikins has is correct. 

There is a separate issue, #433, that covers limiting Namespace create/delete to users where user.is_staff. We'll add the permission checks as part of that issue.

Reply:
Thanks for the clarification.